modules/accessing-inference-endpoint-for-model-deployed-on-single-model-serving-platform.adoc

```diff
@@ -71,7 +91,21 @@ curl --json '{"model_id": "<model_name>", "inputs": "<text>"}' \
 https://<inference_endpoint_url>:443/api/v1/task/server-streaming-text-generation \
 -H 'Authorization: Bearer <token>' <1>
 ----
-<1> You must add the `Authorization` header and specify a token value _only_ if you enabled token authorization when deploying the model.
+<1> Add the `Authorization` header and specify a token value only if you enabled token authorization when deploying the model.
+
+*Caikit Standalone ServingRuntime for KServe*
+
+.HTTP
+[source]
+----
+curl -H 'Content-Type: application/json' -d '{"inputs": "<text>", "model_id": "<model_id>"}' /api/v1/task/embedding
+----
+
+.gRPC
+[source]
+----
+grpcurl -insecure -d '{"text": "<text>"}' -H \"mm-model-id: <model_id>\" <isvc_hostname>:443 caikit.runtime.Nlp.NlpService/EmbeddingTaskPredict
```

Comment:
Can replace `<isvc_hostname>` with `<inference_endpoint_url>`, like we've done for the other runtimes.